### ISSUE TITLE
Add Mac OS X MacPorts installation instructions

### ISSUE THREAD

**Issue by stloeffler** [TeXworks/texworks#716]

**Comment by mojca** [comment]
I only noticed this issue now.

I can help writing the documentation, but it might make sense to do some common brainstorming first.

I would propose to create a single page explaining how to build TeXworks on Mac OS X (pretty much the same way as on Linux) and then two separate pages linked from the main OS X page explaining how to install MacPorts/HomeBrew if that seems necessary.

The HomeBrew instructions are certainly outdated, in particular things like "look for "Xcode 3" (that is only true for OS X 10.6 which is already old/deprecated/unsupported). Or
~~~~~
git clone git://github.com/Sharpie/TeXworks.git`
~~~~~

It would probably be even better if one single page would be devoted to build instructions that work on pretty much any platform. I mean, the following instructions are universal and I find it useless to repeat the same instructions over and over again:
~~~~~
$ git clone https://github.com/TeXworks/texworks.git
$ cd texworks
$ mkdir build
$ cd build
$ cmake ..
$ make
~~~~~

On MacPorts one can install TeXworks with just
~~~~~
sudo port install texworks
~~~~~
It would nevertheless make sense to list all the dependencies that are needed to build TeXworks from source.

**Comment by stloeffler** [comment]
> I would propose to create a single page explaining how to build TeXworks on Mac OS X (pretty much the same way as on Linux)

Agreed. I would not create separate sub-pages, though, as I would assume the MacPorts part would be rather short (and the Homebrew part can probably be shortened as well by being slightly less verbose about what each and every command does). I would propose the following structure:

- MacPorts
  - Requirements/Setup
  - Obtaining & Building TeXworks (the one-liner you mentioned)
  - Updating (probably `sudo port update texworks`?)
- Homebrew
  - Requirements/Setup
  - Obtaining & Building TeXworks
  - Updating

> The HomeBrew instructions are certainly outdated

Yes. I just updated (some of) the links so people would now clone the official repo and go to the new homebrew web page, but the part about Xcode I don't know anything about.

> It would probably be even better if one single page would be devoted to build instructions that work on pretty much any platform

I don't fully agree with this statement. As much as I dislike redundancy (in programming), I think it is important for people to have one guide they can follow from beginning to end without having to jump to other pages/sections, etc. So I would like to keep Linux and Mac separate (on Windows, everything is slightly different anyway because there, additional options are needed, etc.). And since the more tricky/complex/lengthy part often is the "requirements/setup" part and the "update" part, I can live with a little redundancy of 5 lines ;).
